### DISCUSSION TITLE
Codespace connection issues

### DISCUSSION THREAD

**Discussion by KevinSRobinson** [community/community#42398] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi 

Working with code spaces I seem to continually get some sort of  connection issues which means I have to refresh my browser to fix and start my application running again.     This makes development almost impossible.

_These errors seem to constantly occur._
**"Remote Extension host terminated unexpectedly 3 times within the last 5 minutes."
"Failed to save '.....ts': Unable to write file 'vscode-remote://...../.....s'**

**Comment by bdmac** [comment]
Interesting. Mind sharing what geo you're in? Also have you tried using the desktop VS Code client? You might have a better experience that way. https://docs.github.com/en/codespaces/developing-in-codespaces/using-github-codespaces-in-visual-studio-code
